humans/chain.json

```diff
@@ -0,0 +1,257 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "humans",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://humans.ai/",
+  "pretty_name": "humans",
+  "chain_id": "humans_1089-1",
+  "bech32_prefix": "humans",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
Submitted PR: https://github.com/satoshilabs/slips/pull/1597

Reply:
PR merged